reconcile/integrations_manager.py

```diff
@@ -369,15 +399,28 @@ def run(
         logging.debug("Nothing to do, exiting.")
         sys.exit(ExitCodes.SUCCESS)
 
-    ri, oc_map = ob.fetch_current_state(
-        namespaces=namespaces,
-        thread_pool_size=thread_pool_size,
-        integration=QONTRACT_INTEGRATION,
-        integration_version=QONTRACT_INTEGRATION_VERSION,
-        override_managed_types=["Deployment", "StatefulSet", "CronJob", "Service"],
-        internal=internal,
-        use_jump_host=use_jump_host,
-    )
+    fetch_args = {
+        "namespaces": namespaces,
+        "thread_pool_size": thread_pool_size,
+        "integration": QONTRACT_INTEGRATION,
+        "integration_version": QONTRACT_INTEGRATION_VERSION,
+        "override_managed_types": ["Deployment", "StatefulSet", "CronJob", "Service"],
+        "internal": internal,
+        "use_jump_host": use_jump_host,
+    }
+
+    if not image:
+        image = IMAGE_DEFAULT
```

Comment:
do we need defaults? what prevents us from expecting values at all times?

Reply:
Yes, to minimize changes required during the rollout for Fedramp

Reply:
ah understood. lets revisite once this has been rolled out everywhere